packages/web/src/components/buy-audio-modal/components/SuccessPage.tsx

```diff
@@ -66,7 +67,9 @@ export const SuccessPage = () => {
           <span>
             +
             {transactionDetails.status === Status.SUCCESS
-              ? formatAudio(transactionDetails.transactionDetails.change, 0)
+              ? wAUDIO(BigInt(transactionDetails.transactionDetails.change))
+                  .trunc()
+                  .toFixed()
```

Comment:
nit: do we want groupings etc here? if so `toLocaleString` with max 0 decimals is better